CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 ## HEAD
 
+## 0.16.3
+
```

Comment:
Please remove the `0.16.3` header, and keep the changelog addition within HEAD. The way the changelog works is that when I make the next release everything under head will be moved into the release section and a new HEAD section created.

Thanks!

Reply:
done!